src/main/resources/applicationContext-LutungMandrill-all.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd put this to be a part of the README in a section titled how to setup Mandrill DI for Spring, but wouldn't put it as a part of the repo code.